synapse/rest/media/v1/media_repository.py

```diff
@@ -293,7 +293,7 @@ def get_remote_media_info(self, server_name, media_id):
         defer.returnValue(media_info)
 
     @defer.inlineCallbacks
-    def _get_remote_media_impl(self, server_name, media_id):
+    def get_media_from_cache_or_remote(self, server_name, media_id):
```

Comment:
This is no longer a change that is needed, and I'd rather keep private functions private.